src/components/orgunits/OrgUnitData.js

```diff
@@ -1,33 +1,43 @@
+import { useDataEngine } from '@dhis2/app-runtime'
```

Comment:
Could we use useDataQuery (with the the query defined above the component) instead of useDataEngine (meant for non-react contexts)?